services/repository/src/test/java/org/sagebionetworks/auth/services/AuthenticationServiceImplTest.java

```diff
@@ -154,25 +186,6 @@ public void testValidateOAuthAuthenticationCodeEmailNull() throws NotFoundExcept
 		assertEquals(session, result);
 	}
 	
-	@Test
-	public void testValidateOAuthAuthenticationCodeNoMatch() throws NotFoundException{

```

Comment:
try to add this back.

Reply:
Bruce, this test was testing for the case where the AuthenticationService.validateOAuthAuthenticationCodeAndLogin() throws a NotFoundException.  The service no longer throws the NotFoundException since that now happens at the UserManager level.  Therefore, the test no longer serves a purpose, and would not pass.